docs/zh_cn/tutorials/how_to_do_regression_test.md

```diff
@@ -49,51 +68,49 @@ python ./tools/regression_test.py \
 
 ## 例子
 
-1. 测试 mmdet 和 mmpose 的所有 backend 的 转换+精度
+1. 测试 mmdet 和 mmpose 的所有 backend 的 **转换+精度**
 
 ```shell
-python ./tools/regression_test.py \
-    --deploy-yml ./configs/mmdet/mmdet_regression_test.yaml ./configs/mmpose/mmpose_regression_test.yaml \
+python ./regression_test/regression_test.py \
+    --codebase mmdet mmpose \
     --backends all \
     --work-dir "../mmdeploy_regression_working_dir" \
     --device "cuda" \
     --log-level INFO \
     --performance
 ```
 
-2. 测试 mmdet 和 mmpose 的某几个 backend 的 转换+精度
+2. 测试 mmdet 和 mmpose 的某几个 backend 的 **转换+精度**
```

Comment:
The difference between the first and the second command is only '--performance v.s. -p", right? We can just keep the one with '-p'

Reply:
The backend is not the same.
Or may be should I make all the `--performance` to `-p` in the example ?